cmd/osm-injector/reconcile.go

```diff
@@ -27,15 +28,15 @@ func createReconciler(kubeClient *kubernetes.Clientset) error {
 		OsmWebhook:   webhookConfigName,
 		OsmNamespace: osmNamespace,
 	}).SetupWithManager(mgr); err != nil {
-		log.Error().Err(err).Msg("Error creating controller to reconcile MutatingWebhookConfiguration")
-		return err
+		return errors.Wrap(err, "Error creating controller to reconcile MutatingWebhookConfiguration")
```

Comment:
Does `Wrap()` log the message at this point?

Reply:
Yes, that's the whole point of Wrap: https://play.golang.org/p/Aq4sdQMtzah